zerver/tests/test_auth_backends.py

```diff
@@ -5539,10 +5535,10 @@ def test_django_to_ldap_username_with_append_domain(self) -> None:
 
             self.mock_ldap.directory['uid="hamlet@test",ou=users,dc=zulip,dc=com'] = {
                 "cn": ["King Hamlet"],
-                "uid": ['"hamlet@test"'],
+                "uid": ["hamlet@test"],
             }
             username = self.backend.django_to_ldap_username('"hamlet@test"@zulip.com')
-            self.assertEqual(username, '"hamlet@test"')
+            self.assertEqual(username, "hamlet@test")
```

Comment:
I think this behavior change is correct, but we should consider whether it might be a migration issue for someone?